spec/models/login_spec.rb

```diff
@@ -12,7 +12,6 @@
     end
 
     it { is_expected.to validate_presence_of(:identification) }
-    it { is_expected.to validate_uniqueness_of(:identification) }
```

Comment:
If we remove this expectation, there should probably still be a criterion that the identification be unique _scoped to_ a provider.